aws/requests.yaml

```diff
@@ -21,6 +21,8 @@ releases:
   requests:
   - name: coredns
     version: ">= 1.14.2"
+  - name: observability-bundle
```

Comment:
if you add this here, there is no need to keep having the same clause at line 12 and 13 (since > 18.2.1 also includes >= 19.0.0). Please remove either one

Reply:
Well, when you created 18.2.1 you did not take it when it was set for 19.0.0 so I'm not sure what to do here

Reply:
of course not.
if you set it to the `> 18.2.1` block then it's valid for 19.0.0 as well
if you set it to the `> 19.0.0` block, it's NOT valid for 18.2.1.

so you might want to remove it from the `> 19.0.0` block in order for your request to be valid for both 18.2.2+ and 19.0.0+